greet.js

```diff
@@ -0,0 +1,8 @@
+export function greet(name) {
+    
+    if (name === 'Thomas' || name === 'Martin' || name === 'Stefan') return 'Hello Coach!'
+    if (name === '') return 'Hello stranger!' 
+    if (name === undefined) return 'Hello stranger!'
```

Comment:
You could combine these two like line 3
```suggestion
    if (name === '' || name === undefined) return 'Hello stranger!' 
```